grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
```

Comment:
>Since braces aren't listed as keywords in the spec, I don't think they should have a scope name with keyword in them.
>
>The braces are described in the section on Operators and Delimiters, but they are never mentioned in the Operators section. I concluded that they are are not operators, and must therefore be delimiters.

Fair point... So I guess `meta.scope.brace.curly.go` would make sense then...